src/services/store/auth/Slices.ts

```diff
@@ -56,7 +59,9 @@ const appSlice = createSlice({
     checkDeviceError: (state, { payload }) => {
       state.devicePin = payload?.detail || '';
       if (payload.status !== 401) {
-        state.errorString = `${payload.status} - ${payload?.title}`;
+        state.errorString = `${payload.status} - ${
```

Comment:
please, could you change it to          
state.errorString =  payload.status &&
          payload.title ? `${payload.status} - ${payload.title}` : defaultPinError